receiver/kafkametricsreceiver/metadata.yaml

```diff
@@ -40,6 +56,34 @@ metrics:
       value_type: int
       aggregation_temporality: cumulative
     attributes: [topic]
+  kafka.topic.replication_factor:
+    enabled: true
+    description: replication factor of a topic.
+    unit: "1"
+    gauge:
+      value_type: int
+    attributes: [topic]
+  kafka.topic.log_retention_ms:
+    description: log retention period of a topic (ms), Default value is 7 days (604800000ms).
```

Comment:
Since it's a kafka setting, is it valuable to include the default here? I'm worried this may go stale, or just not provide useful information to the user.

Reply:
Sure, Removed that default value from the docs, I am anyways not adding any logic to set these defaults, Kafka takes care of that.